src/data-loaders/RelationalDataLoader.js

```diff
@@ -0,0 +1,64 @@
+import Sequelize from 'sequelize';
+import get from 'lodash/get'
+import Promise from 'bluebird'
+
+export default class RelationalDataLoader {
+  constructor(config) {
+    this.config = config;
+  }
+
+  async load(req) {
+    try {
+      const requiredKeys = [
+        'dbDialect',
+        'dbUsername',
+        'dbPassword',
+        'dbHost',
+        'dbName',
+      ];
+      if (!this.config.rds) {
+        throw new Error('RDS configuration not passed');
+      }
+      const missingKey = requiredKeys.find((key) => {
+        return !this.config.rds[key];
+      });
+      if (missingKey) {
+        throw new Error(`${missingKey} is required.`);
+      }
+      const config = {
+        uri: `${this.config.rds.dbDialect}://${this.config.rds.dbUsername}:${this.config.rds.dbPassword}@${this.config.rds.dbHost}:${this.config.rds.dbPort}/${this.config.rds.dbName}`,
+        options: {
+          host: process.env.DB_HOST,
+          logging: console.log,
+          dialect: process.env.DB_DIALECT,
+          dialectOptions: {
+            decimalNumbers: true,
+            multipleStatements: true,
+          },
+          define: {
+            underScored: true,
+            timestamps: true,
+          },
+        },
+      };
+
+      const sequelize = new Sequelize(config.uri, config.options);
+      const result = await Promise.mapSeries(
+        req.statements,
+        (statement) => {
+          return sequelize.query(statement);
+        },
+      );
+      
+      const rows = get(result, `[1][1].rows`, null)
+      const length = rows?.length;
+      if (length === 1) {
+        return rows[0];
+      }
+      return rows;
```

Comment:
@alichherawalla Sorry, I have no experience with RDS resolvers. 
Does this mean that if there is only one row, it returns one item, and if many an array?

I'm just wondering how this works when you do for example a `getItems` operation that can return more than one row, but  happens to return just one (because there is just one row in DB).

How does that work?